app/models/trainee.rb

```diff
@@ -81,6 +81,10 @@ def dttp_id=(value)
     super
   end
 
+  def full_name
```

Comment:
We already have a `trainee_name` view helper in `TraineeHelper`, and it looks as though this method does the same thing.

Could you reuse the existing code?

Reply:
Good spot.